server/src/command_table_create.go

```diff
@@ -382,7 +390,8 @@ func commandTableCreate(s *Session, d *CommandData) {
 	// Alert the people on the waiting list, if any
 	// (even if they check the "Alert people on the waiting list" checkbox,
 	// we don't want to alert on password-protected games or test games)
-	if t.AlertWaiters &&
+	if preGameVisible &&
```

Comment:
afaik this condition will never be met, so it is probably pointless

Reply:
No, in the normal case (user creates a table) preGameVisible is true

Reply:
allow me to clarify. by design, the waiting list will always be empty when the code reaches this point. thus, your added check doesn't do anything really